.github/workflows/syntax.yml

```diff
@@ -1,5 +1,8 @@
 name: Run syntax tests
 
+permissions:
+  contents: read
```

Comment:
Why do we need `read` here? For cloning?

Reply:
That is my understanding, but let me try removing it and run it without, to confirm that theory.

Reply:
Looks like that is not needed, so I'll also remove it from the test workflow and review the same permission in LS.

Good catch - thank you!